rails/app/controllers/api/v1/report_learners_es_controller.rb

```diff
@@ -104,6 +104,7 @@ def external_report_query_jwt
           id: url_for(current_user),
           email: current_user.email
         },
+        portal_url: root_url,
```

Comment:
You probably need to get this from an environment variable since on learn-report.concord.org you probably want learn.concord.org instead of learn-report.concord.org